code/__defines/mobs.dm

```diff
@@ -2,6 +2,7 @@
 #define CONSCIOUS   0
 #define UNCONSCIOUS 1
 #define DEAD        2
+#define FUGUE       4
```

Comment:
I'm not sure if there are any `stat >= DEAD` left, but it may not be intentional for fugue to be counted as "deader than dead".
Pre-posting edit: There are several instances of stat > which would probably be broken by `FUGUE > DEAD`. Since it's not exactly in-scope to change all of those, I'm not sure what to do about this.